contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,536 @@
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Watch real users use your product; diagnose bugs</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Heatmaps</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Visualize where users click in your app or website</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Feature flags</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Roll out features safely; toggle features for cohorts or individuals</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Surveys</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Ask users for qualitative feedback and gather responses</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>A/B testing</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Test changes and analyze impact</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Apps/Integrations</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Push and pull data to other destinations</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Open source</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Full access to source code and transparent roadmap</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Analytics features comparison
```

Comment:
I think a big one GA does a good job of that we don't (that is missing) is monetization. In-app purchases, purchase conversion, etc. When data warehouse is complete, maybe we revisit this:
 
![Screen Shot 2023-10-16 at 1 17 58 PM](https://github.com/PostHog/posthog.com/assets/34755028/e1dcbee4-3387-46be-8354-022137a4b6ee)